content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -85,8 +85,6 @@ different Kubernetes components.
 | `CSIMigrationGCE` | `false` | Alpha | 1.14 | 1.16 |
 | `CSIMigrationGCE` | `false` | Beta | 1.17 | 1.22 |
 | `CSIMigrationGCE` | `true` | Beta | 1.23 | |
-| `CSIMigrationOpenStack` | `false` | Alpha | 1.14 | 1.17 |
-| `CSIMigrationOpenStack` | `true` | Beta | 1.18 | |
```

Comment:
These two lines should be migrated into the second table rather than simply dropped from the first  table. I.e. please move `CSIMigrationOpenStack` to somewhere around line 264 and mark it as GA.